docs/how_tos/how_to_support_new_devices.md

```diff
@@ -28,7 +28,13 @@ The next step is the to add an entry of your device to `node_modules/zigbee-herd
 
 ```js
 {
-    zigbeeModel: ['lumi.sens'], // The model ID from: Device with modelID 'lumi.sens' is not supported.

```

Comment:
The `zigbeeModel` is the preferred way of adding a device and is the easiest one. I think we should not mention fingerprint here but only in the how_to_support_new_tuya_devices

Reply:
Fixed